h-util-ui/prisma/migrations/20240915221806_add_words_parsed/migration.sql

```diff
@@ -0,0 +1,18 @@
+-- RedefineTables
```

Comment:
Took way too many migrations due to not taking setting up the schema that seriously 🤡 

May be some way to compress